libnmstate/nm/device.py

```diff
@@ -285,7 +285,7 @@ def _delete_device_callback(src_object, result, user_data):
             mainloop.execute_next_action()
         else:
             mainloop.quit(
-                'Device deletion failed on {}: error={}'.format(
+                'Device deletion failed on {}: unexpected error={}'.format(
```

Comment:
Seems to be an unrelated change.

Reply:
Is good to debug the difference of not matching exception instead of exception that we don't catch.

Reply:
Good argument, but please propose it through its own PR with this reasoning in the message.